docs/getting-started/pingfederate.md

```diff
@@ -0,0 +1,170 @@
+
+## Determine credentials that are able to configure the server
+
+The configuration API connection established by the provider uses basic authentication. The provider will need the username and password of a user with permission to manage server configuration.
+
+In the PingFederate Docker image, the administrative user defaults to the username `administrator` with password `2FederateM0re`.
+
+## Determine what version of PingFederate you are running
+
+The provider requires that the version of PingFederate is specified.  
+
+You can do this one of two ways:
+
+* Using the `product_version` attribute when configuring the provider:
+
+```shell
+provider "pingfederate" {
+  ...
+  product_version = "12.0.0"
+}
+```
+
+* Setting the `PING_PRODUCT_VERSION` environment variable to the version of PingFederate you are running:
+
+```shell
+export PING_PRODUCT_VERSION=12.0.0
+```
+
+If using the container, you can view the product version using by running a shell in the container and searching for the appropriate environment variable:
+
+```shell
+docker container exec -it pingfederate_terraform_provider_container /bin/sh
+
+# In the container shell, get the version
+env | grep PING_PRODUCT_VERSION
+```
+
+!!! note "Precedence"
+    The `product_version` attribute takes precedence over the `PING_PRODUCT_VERSION` environment variable.
+
+If neither is set, the provider will throw an error.
+
+## Trusting PingFederate certificates
+
+PingFederate generates a self-signed certificate by default, which is presented by the server when connecting.
+The default self-signed certificate can also be replaced with a custom certificate. The provider has a few ways of configuring trust for the HTTPS connection with the server.
+
+By default, the provider will trust the host's default root CA set when connecting to the server.
+
+If you need to provide CA certificates for the provider to trust, you can use the `ca_certificate_pem_files` attribute. This attribute supports providing a set of paths to files containing PEM-encoded CA certificates to be trusted.
+
+The `PINGFEDERATE_PROVIDER_CA_CERTIFICATE_PEM_FILES` environment variable can also be used, with commas to delimit multiple PEM file paths if necessary.
+
+Finally, the provider supports an `insecure_trust_all_tls` boolean attribute that enables it to trust all certificates when connecting to the server.
+
+!!! warning "Insecure Trust All TLS"
+    The `insecure_trust_all_tls` flag, when set to `true`, is insecure and is intended for development and testing only.  You should not enable this option for production use.
+
+## Use the provider to configure PingFederate
+
+You are now ready to configure the PingFederate server using the provider.  For example:
+
+```shell
+terraform {
+  required_version = ">=1.4"
+  required_providers {
+    pingfederate = {
+      version = "~> 0.0.1"
```

Comment:
suggest using `version = ">= 0.1.0, < 1.0.0"` so the reader is downloading the latest version that matches the provider schema in the doc